composer.json

```diff
@@ -40,7 +40,8 @@
         "burgov/key-value-form-bundle": "to edit assoc multivalue properties. require version 1.0.*"
     },
     "conflict": {
-        "phpcr/phpcr-shell": "<1.0.0-beta1"
+        "phpcr/phpcr-shell": "<1.0.0-beta1",
+        "symfony/framework-bundle": ">=2.6.0,<2.6.6"
```

Comment:
is this the correct way to specify we conflict with 2.6 versions < 2.6.6 but 2.3.27 is ok?

what about 2.4 and 2.5? should we start to conflict with those?